templates/zerver/realm_creation_disabled.html

```diff
@@ -0,0 +1,12 @@
+{% extends "zerver/portico.html" %}
+
+{% block for_you %} got a bit lost there. {% endblock %}
+
+{% block portico_content %}
+
+<br/>
+<p class="lead">Realm creation disabled</p>
+
+<p>This service is not taking new customer teams. Zulip is open source, so you can install your own Zulip server by following the instructions on www.zulip.org!</p>
```

Comment:
s/This service/This server/.

Would make the two sentences separate paragraphs, and also tag for translation (see my comment linking to how to do that below)